resource_obmcs_identity_compartment_test.go

```diff
@@ -53,11 +53,11 @@ func (s *ResourceIdentityCompartmentTestSuite) TestAccResourceIdentityCompartmen
 			{
 				Config: s.Config + `
 				resource "oci_identity_compartment" "t" {
-					name = "-tf-compartment"
+					name = "-tf-compartment2"
 					description = "tf test compartment2"
 				}`,
 				Check: resource.ComposeTestCheckFunc(
```

Comment:
This would have passed before the rename changes, right? Should check that the ID in fact stays the same.

Reply:
Updated